src/static/pretixcontrol/js/ui/main.js

```diff
@@ -143,7 +143,10 @@ $(function () {
                 clear: 'fa fa-trash',
                 close: 'fa fa-remove'
             }
-        });
+        };
+        if ($(this).data().hasOwnProperty("paymentWeekdaysDisabled"))
```

Comment:
I find jQuery's ``data()`` thing not very intuitive, how about ``$(this).is('[data-is-payment-date]')`` instead to keep with the HTML notation?